packages/widgetbook/lib/src/app/widgetbook_shell.dart

```diff
@@ -17,8 +18,10 @@ class WidgetbookShell extends StatelessWidget {
 
     return ColoredBox(
       color: Theme.of(context).colorScheme.surface,
-      child: Row(
-        crossAxisAlignment: CrossAxisAlignment.stretch,
+      child: ResizableWidget(
```

Comment:
@Kiruel I modified this code a bit as follows:
1. Sort `children` to be the last parameter for more readability.
2. Remove `Material` widget; You probably ran into this issue because you've used the hosted version of `widgetbook_core` and not the local one. We use [melos](https://melos.invertase.dev/) to manage dependencies, so consider using `melos bs` instead of `flutter pub get` in your future PRs.